### ISSUE TITLE
Goal/Task to generate a Devfile

### ISSUE THREAD

**Issue by manusa** [eclipse/jkube#1476]
### Description

As a user, I want to be able to transition to other tools, so that I can easily move away from JKube or be able to work on my project from different environments.

A solution for this is to generate a [Devfile](https://github.com/devfile) based on JKube's settings and its inferred project configuration.

This would allow users to other tooling such as [odo](https://github.com/redhat-developer/odo), and/or remote development environments such as [Eclipse Che](https://www.eclipse.org/che/).

**Comment by manusa** [comment]
Not a priority right now, we'll wait on feedback from the community to see if this is really a valuable enhancement.
